apps/mobile-alphabet/src/app/Components/Background.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { StyleSheet, useWindowDimensions, View } from 'react-native';
+import Svg, { Defs, LinearGradient, Rect, Stop } from 'react-native-svg';
+import theme from './../../../theme.config.json';
+
+const FROM_COLOR = theme.colors.primary;
+const TO_COLOR = theme.colors.secondary;
+
+const Background = ({ children }) => {
+    return (
+        <View style={{ flex: 1 }}>
+            <Svg
+                height={useWindowDimensions().height}
```

Comment:
You should probably call this once above line 10:
const {height,width} = useWindowDimensions();

Reply:
Changed.